INSTALL.md

```diff
@@ -2,65 +2,61 @@
 
 Requirements are added in requirement.txt file
 
-python & packages
-----------------
-
-`Python 3.7.3 - 64 bit`
+Python 3.7.3 
 
 python:
-install python for the respective OS at `https://www.python.org/downloads/` Make sure to update the path variable to point to the python installation folder.
+install python for the respective OS at https://www.python.org/downloads/ Make sure to update the path variable to point to the python installation folder.
 
 pip:
-get `get-pip.py` from below link to your folder `https://bootstrap.pypa.io/get-pip.py` Open a command prompt and navigate to the folder containing `get-pip.py`. Run the following command:
+get get-pip.py from below link to your folder https://bootstrap.pypa.io/get-pip.py Open a command prompt and navigate to the folder containing get-pip.py. Run the following command:
 
 pandas:
-`python -m pip install pandas`
+python -m pip install pandas
+
+xlsxwriter:
+python -m pip install xlsxwriter
+
+xlrd:
+python -m pip install xlrd
 
 pylint:
-`python -m pip install -U pylint`
+python -m pip install -U pylint
 
 mutmut:
-`python -m pip install mutmut`
+python -m pip install mutmut
 
 pytest:
-`python -m pip install pytest`
+python -m pip install pytest
 
-unittest:
-`python -m pip install unittest`
+unit test:
+python -m pip install unit test
 
 codecov:
-`python -m pip install codecov`
+python -m pip install codecov
 
 pytest-cov:
-`python -m pip install pytest-cov`
+python -m pip install pytest-cov
 
 Other tools
 -----------
```

Comment:
You could consider adding md linting.

Reply:
Added and addressed all md file lint errors.